src/main/java/com/oroarmor/multiitemlib/api/ICustomShieldDisableCooldown.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2021 OroArmor (Eli Orona)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.oroarmor.multiitemlib.api;
+
+public interface ICustomShieldDisableCooldown {
+
+    /**
+     * If the shield is disabled by an axe, how long should the cooldown last for?
+     * Set to 0 to make the shield never suffer from being disabled by an axe. (Other shields in your inventory still get disabled)
+     * Default value is 100.
+     */
+    int getShieldCooldown();
```

Comment:
hm, not sure how hard it would be but passing in the item stack for the shield and maybe the disabling attack would be nice. Looks like the method where you call this has those available as the 3rd and 2nd parameters respectively.

Reply:
I took a look at trying this, but both funcs I mix into do not use an ItemStack. Since it's just disabling Items.SHIELD we're basically just giving it the static item instantiations in the same way. (Check Items.SHIELD, no cooldown so we make it 100, pass over a custom shield so we give it the value, etc) there's no stack to pass in unfortunately, or I'd've liked to do this tbh